interview-questions.md

```diff
@@ -8,44 +8,66 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What does calling super do in a Ruby class?
 
-Your answer:
+Your answer:a super method is a part of ruby inheritence . it pulles over fromthte parent class what it wants kinda like going to you moms house and taking for toliet paper
+
+Researched answer: super gives you acess to the parent classes attribute. it ca be called with an arugument which will add to the parameter of the function or it can have no arguments and just use the attribute provided by the parent function .
+
+
+
+
+2. What is a gem? 
+
+Your answer: gem is a file folder in ruby , kinda like json package is for javascript.
+
+Researched answer: a gem is a packaged library or application that can be shared and installed. It is a set of reusable code and functionality that can be included in other Ruby programs. you have public an private gems as well. private gems are found in enterprise envirenments.
```

Comment:
I encourage you to use examples, particularly for questions like this where you can talk about your experience with a gem and how you implemented it.